utils/mock.js

```diff
@@ -37,29 +35,30 @@ function mockQuery (schema, mockedSchema, parsedQuery, fixture, saveFixture, glo
       // Check if the query receives args and check if the required ones are passed
       argumentsValidator(queryArgs, querySchema[0].arguments, name, queryVariables)
       // If there are fixtures, set the values
-      if (fixture && fixture.data) {
+      if (fixture && fixture.data !== undefined) {
         mock = setFixture(mock, fixture.data, name)
         if (saveFixture) {
           mockedSchema[Query][name] = mock
         }
       }
       // Return the mock of the selected fields
-      mockedQuery = { [queryName]: mock }
-      if (globalQueryVariables) {
-        return {
-          globalQueryVariables: queryVariables,
-          mockedQuery
-        }
-      }
-      return mockedQuery
+      return Object.assign(
+        {
```

Comment:
What do you think if we change in all the places this validation for this:
```js
return response(queryName, mock, errors, globalQueryVariables, queryVariables)   
```
and we create a new method that will be
```js
function response(queryName, mock, errors, globalQueryVariables, queryVariables) {
  return Object.assign(
    {
      mockedQuery: Object.assign(
        {},
        mock === undefined ? undefined : { data: { [queryName]: mock } },
        errors ? { errors } : undefined
      )
    },
    globalQueryVariables ? { globalQueryVariables: queryVariables } : undefined
  )
}
```

Reply:
@estrada9166  could you please apply the changes directly in the PR ?

Reply:
Sure thing!!

Reply:
thanks !